### PR TITLE
[#160551023] app end points fixed

### PR DESCRIPTION
**Pull request by TeamoreA**
**What does this PR do?**

- shows all orders (GET request)
- shows a specific order specify using its ID(GET request)
- creation of a new order (POST request)
- update an existing order (PUT request)
- deletion of an order (DELETE request)

**Description of Task to be completed?**

- add the endpoint to list all orders
- add an endpoint to get a specific order
- add an end point to create a new order
- add an endpoint to update an existing order
- add an endpoint to delete an order

**How it can be manually tested?**

- install the dependencies in requirements.txt
- clone the project https://github.com/TeamoreA/fast-food-apiV1/ and run the orders.py
- Test the various URLs in POSTMAN

**What are the relevant pivotal tracker stories?**
- Create new [#160533576](https://www.pivotaltracker.com/n/projects/2198040)
- List all [#160533562](https://www.pivotaltracker.com/n/projects/2198040)
- List one [#160533566](https://www.pivotaltracker.com/n/projects/2198040)
- Update [#160533577](https://www.pivotaltracker.com/n/projects/2198040)
- Delete [#160533580](https://www.pivotaltracker.com/n/projects/2198040)

**Screenshots
screenshot 66
screenshot 67
screenshot 68
screenshot 69
screenshot 70

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export FLASK_APP = run.py
+export SECRET = 'THISISSECRET'
+export APP_SETTINGS = "development"
+export DATABASE_URI = ""
```
Comment (mwaz) [inline .env]: add a new line at the end of this file

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+install:
+  - pip install -r requirements.txt
+  - pip install coveralls
+
+script:
+  - nosetests --with-coverage --cover-erase --cover-package=app/test
+
+after_script:
+  - coveralls
```
Comment (mwaz) [inline .travis.yml]: add a new line at the end of this line

File: Procfile [hunk 3]
```diff
@@ -0,0 +1 @@
+web: gunicorn run:APP
```
Comment (mwaz) [inline Procfile]: add a new line at the end of this line

Comment (TeamoreA) [reply Procfile]: am i adding blank lines. i didn't get clearly please?

**Pull request merged**